yt/yt/client/table_client/composite_compare.cpp

```diff
@@ -148,6 +150,36 @@ Y_FORCE_INLINE static int CompareYsonItems(const TYsonItem& lhs, const TYsonItem
     return ComparePrimitive(static_cast<ui32>(lhsClass), static_cast<ui32>(rhsClass));
 }
 
+// Returns the minimum binary size that should be accounted for with this item.
+// Item should not be Map or Attribute-related, there is no special handling for them.
+i64 GetMinResultingSize(const TYsonItem& item, bool isInsideList)
```

Comment:
Actually it's hard to understand what does this function do even with comment.

Reply:
Tried to improve it.